static/register.html

```diff
@@ -10,13 +10,117 @@
     <!-- RegExp.escape() is used by the "confirm password" field -->
     <script crossorigin="anonymous" src="https://benjamingr.github.io/RegExp.escape/polyfill.js"></script>
 
+    <!-- Dependencies -->
+    <script type="text/javascript" src="/min/modernizr-min.js"></script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+
     <script>
         /* Functions */
         function getToken() {
             var query = new URLSearchParams(window.location.search)
             return query.get("token")
         }
 
+        // Passes a user object to callback TODO consider using Promises (what's the oldest browser that can support them?)
+        /*
+            {
+                id: the user's id
+                email: the user's account email
+                username: the user's username (not nickname)
+                tag: the user's username#discriminator tag
+                avatar: the user's avatar url (PNG or GIF)
+                nitro: whether or not the user has nitro
+            }
+         */
+        function getDiscordUser(access_token, callback) {
+            // Materializecss needs jQuery so we may as well use it too
+            $.get({
+                url: "https://discordapp.com/api/v6/users/@me",
```

Comment:
lol imagine being on v6

Reply:
imagine the previous versions being completely disabled/deleted